Avali Species Doc (#578)
space-wizards/docs: src/en/space-station-14/character-species/proposals/avali.md

# Avali Species Document

| Designers | Coders | Implemented | GitHub Links |
|---|---|---|---|
| NoreUhh | NoreUhh + TBD | :x: No | TBD |

## Overview

This proposal document aims to cover the Avali as a playable round-start species.

> The Avali are a small-statured, "silly" bird-like alien species with a very distinctive look. With their 4 big ears, large feathered arms/wings, and feathered tails, they stand out from other species easily at a glance. While the Avali don't have any unique "abilities", the Avali still have many traits that help them stand out easily from other species, and that causes unique interactions to happen amongst players.

Reviewer: ```suggestion
The Avali are a small, bird-like, "silly"- looking alien species. With their four big ears, large feathered arms/wings, and feathered tails, they stand out from other species easily at a glance. While the Avali don't have any unique "abilities", they still have many traits that help them stand out from other species, which leads to unique interactions among players.
```

## Background

The Avali are a very common round-start species on many downstream forks. As a result, the Avali have become a popular species choice for many players. This has led to the discussion of them being added to upstream time and time again.

> The Avali can be found on many popular forks such as Starlight, RMC-14, and Harmony, just to name a few. However, each fork's iteration of Avali can differ slightly from the others. The goal of this document is to incorporate unique additions to Avali across different forks and combine them, as well as implement some of our own unique additions, aiming to strengthen upstream by providing new and interesting interactions between players and a new species that players can enjoy thematically.

Reviewer: ```suggestion
The Avali is available on many popular forks, including Starlight, RMC-14, and Harmony. However, each fork's iteration of Avali can differ slightly from the others. The goal of this document is to incorporate unique additions to Avali across different forks and combine them, and to implement some of our own additions, aiming to strengthen upstream by providing new and interesting player interactions with a new species that players can enjoy thematically.
```

## Features/Design

The Avali stand out as a mostly unique species. While they do share similar "avian" features to the Vox, they differ drastically enough to be a completely separate species from them, especially in regard to their mechanics differing greatly from one another.

### Core Visual Elements

Avali are fuzzy, bipedal creatures. The main identifying features of Avali are their 4 giant ears, large feathered wings, and tails. Avali stand digitigrade and communicate with others using their unique squeaky voices. Like other species with tails, they protrude from their jumpsuits/hardsuits, making them easy to identify. The same can be said of their large, winged feathers, which make them hard to miss. Avali are normally small-statured, but the size of their wings and feathers makes them seem larger than they really are, like an animal with a puffy coat of fur.

### Naming Convention

Avali have very simple naming conventions. Avali most commonly have singular "exotic" names. However any naming schematic can fit their species.

> Some examples of Avali names include: Narodi, Kalomi, Eijuli, Eilan, and Maaka

Reviewer: ```suggestion
Some examples of Avali names include: Narodi, Kalomi, Eijuli, Eilan, and Maaka.
```

### Features Of The Species

#### Preference To Cold

> The Avali have a naturally lower body temperature compared to other species. This means they prefer colder environments, giving them slight resistance to cold, and that their acceptable temperature range is lower than that of other species. This, along with the fuzz on their bodies, makes them more susceptible to heat damage.

Reviewer: ```suggestion
The Avali have a naturally lower body temperature compared to other species. This means they prefer colder environments, exhibit slight cold tolerance, and have a lower acceptable temperature range than other species. This, along with the fuzz on their bodies, makes them more susceptible to heat damage.
```

> The Avali have a naturally lower body temperature compared to other species. This means they prefer colder environments, giving them slight resistance to cold, and that their acceptable temperature range is lower than that of other species. This, along with the fuzz on their bodies, makes them more susceptible to heat damage.

Reviewer 2: How cold specifically? Starlight has them at ~230k. I personally think this is much to cold as it makes no sense to be able to maintain that temperature when its around 310k on the station.

Author: Humans as a baseline are around 308k or 35C
Avali are roughly at 296k or 23C

This isn't something that is really set in stone either. IMO it's more reasoning to give them the resistance to cold / susceptibility to heat thing without just being like "oh yeah they got fur so they get fried easily" as the only reason.

I agree though, 230K would be way too low to make sense

#### Faster Metabolism

One of the more unique differences between Avali and other species is the speed of their metabolism. In contrast to Arachnids, their metabolism is slightly faster instead of slightly slower. This creates a "double-edged sword" situation for Avali players and those who interact with them. For example, Avali will now heal faster from injected medicines, meaning they could potentially recover more quickly from a fight than other species. However, this comes with its downsides...

Avali are more susceptible to poisoning, as any poison injected into them will be metabolized faster than in the average species. Not only are poisons faster acting on Avali, but overdoses can easily be more lethal. Beneficial chemicals such as Hyperzine/Stimulants will be less effective in Avali patients, as they only produce effects while the user is metabolizing those reagents. The faster a stimulant metabolizes, the less "effective time" the user gets from those chemicals.

> When treating Avali patients, medical staff should take into account the faster metabolism of Avali, as they currently do with Arachnids.

Reviewer: ```suggestion
When treating Avali patients, medical staff should account for Avali's faster metabolism, as they currently do for Arachnids.
```
